pybuildkite/artifacts.py

```diff
@@ -0,0 +1,24 @@
+from pybuildkite.client import Client
+
+
+class Artifacts(Client):
+
+    def __init__(self, client, base_url):
+        self.client = client
+        self.base_url = base_url + "organizations/{}/pipelines/{}/builds/{}/"
```

Comment:
We're calling this parameter `self.path` in other classes. Let's be consistent

Reply:
Wow, that's true. Sorry xD.

And yes, i will add docstring. :+1: